proxy/src/main/java/net/md_5/bungee/BungeeCord.java

```diff
@@ -387,88 +402,118 @@ public void stop()
     @Override
     public synchronized void stop(final String reason)
```

Comment:
Can probably remove synchronized now?

Reply:
Good call - this is indeed not needed anymore